privacyidea/config.py

```diff
@@ -4,6 +4,27 @@
 basedir = "/".join(basedir.split("/")[:-1]) + "/"
 
 
+pubtest_key = b"""-----BEGIN PUBLIC KEY-----
```

Comment:
Since this data is only used in the JWT testcase, i think we should put it in the corresponding test file. I'll fix that.